src/main/scala/loamstream/apps/IndexFiles.scala

```diff
@@ -0,0 +1,69 @@
+package loamstream.apps
+
+import java.io.FileWriter
+import java.nio.file.Path
+
+import org.apache.commons.csv.CSVFormat
+import org.apache.commons.csv.CSVPrinter
+
+import loamstream.conf.ExecutionConfig
+import loamstream.model.jobs.Execution
+import loamstream.model.jobs.LJob
+import loamstream.util.CanBeClosed
+import loamstream.util.Files
+
+/**
+ * @author clint
+ * Jun 4, 2019
+ */
+object IndexFiles {
+  def writeIndexFiles(
+    executionConfig: ExecutionConfig,
+    jobsToExecutions: Map[LJob, Execution]): Unit = {
+
+    def makePath(fileName: String) = executionConfig.logDir.resolve(fileName)
+
+    Files.createDirsIfNecessary(executionConfig.logDir)
+    
+    writeDataTo(makePath("all-jobs.tsv"))(jobsToExecutions)
+
+    import loamstream.util.Maps.Implicits._
+    
+    writeDataTo(makePath("failed-jobs.tsv"))(jobsToExecutions.filterValues(_.isFailure))
+  }
+  
+  private def writeDataTo(file: Path)(executionTuples: Iterable[(LJob, Execution)]): Unit = {
+    val csvFormat = CSVFormat.DEFAULT.
+        withHeader("JOB_ID", "JOB_NAME", "JOB_STATUS", "EXIT_CODE", "JOB_DIR").
+        withDelimiter('\t').
+        withRecordSeparator('\n')
```

Comment:
Okay, totally pedantic, but isn't it more standard practice to put the `.` at the start of each line?

```scala
    val csvFormat = CSVFormat.DEFAULT
        .withHeader("JOB_ID", "JOB_NAME", "JOB_STATUS", "EXIT_CODE", "JOB_DIR")
        .withDelimiter('\t')
        .withRecordSeparator('\n')
```

Reply:
I don't know what's more standard.  There were some weird corner cases with Scala's semi-colon inference at one point which got me in the habit of the trailing `.` - and trailing operators in general - to be extra-clear to the compiler that this is a multi-line expression by making each line invalid on its own.  Maybe those cases are gone now, I don't know.